test/controllers/update/overseas.entity.change.controller.spec.ts

```diff
@@ -121,44 +128,43 @@ describe("Overseas entity do you want to change your OE controller", () => {
     });
 
     test(`redirect to ${WHO_IS_MAKING_UPDATE_URL} on YES selection`, async () => {
-      mockGetCompanyPscService.mockReturnValue(MOCK_GET_COMPANY_PSC_ALL_BO_TYPES);
-      mockGetCompanyOfficers.mockReturnValue(MOCK_GET_COMPANY_OFFICERS);
       mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_MOCK);
-      mockGetCompanyProfile.mockReturnValueOnce(companyProfileQueryMock);
       const resp = await request(app).post(UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL)
-        .send({ [NoChangeKey]: "1" });
+        .send({ [NoChangeKey]: "0" });
```

Comment:
I noticed this test is for the change journey but the data being passed is for the no change journey (NoChnageKey=true), hence the updates I've made